### ISSUE TITLE
mouseover events causing excessive delay

### ISSUE THREAD

**Issue by nickdesaulniers** [honestbleeps/Reddit-Enhancement-Suite#5117]
- My browser (copy from https://www.whatismybrowser.com): Chrome 74 on Windows 10
- My RES version: 5.16.7
- Am I in reddit beta (yes/no): I think so

*What's the problem?*: Excessive delays in mouseover thumbnails after chrome open for a few days, or when scrolling a few pages in Reddit.

*Are there any related posts in https://github.com/honestbleeps/Reddit-Enhancement-Suite/issues or /r/RESissues?*: No.

*What other browser extensions are installed?* uBlock origin, HTTPS everywhere.

attaching chrome devtools profile, which shows 6s spent in foreground.entry.js:914 onmouseover.  Looks like that file is from RES?

[Profile-20190520T193518.json.txt](https://github.com/honestbleeps/Reddit-Enhancement-Suite/files/3200491/Profile-20190520T193518.json.txt)

**Comment by larsjohnsen** [comment]
The profile appears to show that 1.21 ms in the function itself, and the rest is spent on a `Event` `Task`. I have no idea on how to debug that, sorry.

**Comment by nickdesaulniers** [comment]
Yes; sorry, it may be an issue in Chrome itself (not sure how to differentiate).  Disabling RES seemed to make an immediate relief, but I should report back in a few days if the issue is back.  Figured I'd file a bug for future travelers, or in case RES developers could interpret more of the tea leaves.
